$NetBSD: patch-requirements__test.txt,v 1.4 2018/07/04 03:43:12 adam Exp $

'coverage' is not (yet) needed for testing.
Relax pytest requirements.

--- requirements_test.txt.orig	2018-06-17 20:17:58.000000000 +0000
+++ requirements_test.txt
@@ -1,3 +1,2 @@
-coverage>=4.0,<4.4  # TODO: https://github.com/benoitc/gunicorn/issues/1548
-pytest==3.2.5  # TODO: upgrade to latest version requires drop support to Python 2.6
-pytest-cov==2.5.1
+pytest>=3.2.5  # TODO: upgrade to latest version requires drop support to Python 2.6
+pytest-cov>=2.5.1
